lib/compiler-cranelift/src/compiler.rs

```diff
@@ -419,11 +418,11 @@ fn mach_reloc_to_reloc(module: &ModuleInfo, reloc: &MachReloc) -> Relocation {
 }
 
 fn mach_trap_to_trap(trap: &MachTrap) -> TrapInformation {
-    let &MachTrap {
-        offset,
-        srcloc: _,
-        code,
-    } = trap;
+    let &MachTrap { offset, code } = trap;
+    eprintln!(
+        "mach_trap_to_trap({:?}) offset={:?}, code={:?}",
+        trap, offset, code
+    );
```

Comment:
Why eprintln!? Can we remove this?

Reply:
Oops, debug leftover! sorry, I'll remove it :O